server/src/main/clients/web-browser/src/pages/projects/Overview/Overview.tsx

```diff
@@ -1,10 +1,21 @@
 import './Overview.scss';
 import {DefaultPage} from '../../../libs/DefaultPage/DefaultPage';
+import {ProjectDashboard} from '../../../libs/projects/ProjectDashboard/ProjectDashboard';
+import {Layout} from 'antd';
+import {Content} from 'antd/es/layout/layout';
```

Comment:
import {Content} from 'antd/es/layout/layout';
->
const Content = Layout.Content;

Generally, we only want to import 'antd'. Things under 'antd' probably have some other reference.